### PR TITLE
New version: Fndroid.ClashForWindows version 0.20.24

### PR DESCRIPTION
**Pull request by Fndroid**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108180)

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
@russellbanks incorrect installer urla with architecture.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/f/Fndroid/ClashForWindows/0.20.24/Fndroid.ClashForWindows.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Platform:
+- Windows.Desktop
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2023-05-26
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/f/Fndroid/ClashForWindows/0.20.24/Fndroid.ClashForWindows.installer.yaml:15]:
```suggestion
- Architecture: x64
```

File: manifests/f/Fndroid/ClashForWindows/0.20.24/Fndroid.ClashForWindows.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  Scope: user
+  InstallerUrl: https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.20.24/Clash.for.Windows.Setup.0.20.24.exe
+  InstallerSha256: 0F98F971813DFA5BAF711A2F68643C802AD63D0FEBE61A901F62F90311267B4B
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86
```
Comment (vedantmgoyal2009) [inline manifests/f/Fndroid/ClashForWindows/0.20.24/Fndroid.ClashForWindows.installer.yaml:21]:
```suggestion
- Architecture: x64

```

File: manifests/f/Fndroid/ClashForWindows/0.20.24/Fndroid.ClashForWindows.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  Scope: machine
+  InstallerUrl: https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.20.24/Clash.for.Windows.Setup.0.20.24.exe
+  InstallerSha256: 0F98F971813DFA5BAF711A2F68643C802AD63D0FEBE61A901F62F90311267B4B
+  InstallerSwitches:
+    Custom: /ALLUSERS
+- Architecture: x64
```
Comment (vedantmgoyal2009) [inline manifests/f/Fndroid/ClashForWindows/0.20.24/Fndroid.ClashForWindows.installer.yaml:27]:
```suggestion
- Architecture: x86
```

File: manifests/f/Fndroid/ClashForWindows/0.20.24/Fndroid.ClashForWindows.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  Scope: user
+  InstallerUrl: https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.20.24/Clash.for.Windows.Setup.0.20.24.arm64.exe
+  InstallerSha256: 99F3E8DD820F3B77AA2B81CAB29BA98A286C5694D343EB2C67790BDE30A5B307
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x64
```
Comment (vedantmgoyal2009) [inline manifests/f/Fndroid/ClashForWindows/0.20.24/Fndroid.ClashForWindows.installer.yaml:33]:
```suggestion
- Architecture: x86

```

File: manifests/f/Fndroid/ClashForWindows/0.20.24/Fndroid.ClashForWindows.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  Scope: machine
+  InstallerUrl: https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.20.24/Clash.for.Windows.Setup.0.20.24.arm64.exe
+  InstallerSha256: 99F3E8DD820F3B77AA2B81CAB29BA98A286C5694D343EB2C67790BDE30A5B307
```
Comment (vedantmgoyal2009) [inline manifests/f/Fndroid/ClashForWindows/0.20.24/Fndroid.ClashForWindows.installer.yaml:36]:
```suggestion
  InstallerUrl: https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.20.24/Clash.for.Windows.Setup.0.20.24.ia32.exe
  InstallerSha256: ee585e205c19e19e0bb7bf0f51a2791a24cabff7fd045f68c46eb867d26bfe82
```

File: manifests/f/Fndroid/ClashForWindows/0.20.24/Fndroid.ClashForWindows.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.20.24/Clash.for.Windows.Setup.0.20.24.exe
+  InstallerSha256: 0F98F971813DFA5BAF711A2F68643C802AD63D0FEBE61A901F62F90311267B4B
+  InstallerSwitches:
+    Custom: /ALLUSERS
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.20.24/Clash.for.Windows.Setup.0.20.24.arm64.exe
+  InstallerSha256: 99F3E8DD820F3B77AA2B81CAB29BA98A286C5694D343EB2C67790BDE30A5B307
```
Comment (vedantmgoyal2009) [inline manifests/f/Fndroid/ClashForWindows/0.20.24/Fndroid.ClashForWindows.installer.yaml:30]:
```suggestion
  InstallerUrl: https://github.com/Fndroid/clash_for_windows_pkg/releases/download/0.20.24/Clash.for.Windows.Setup.0.20.24.ia32.exe
  InstallerSha256: ee585e205c19e19e0bb7bf0f51a2791a24cabff7fd045f68c46eb867d26bfe82
```

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**